packages/api/src/platforms/vtex/resolvers/product.ts

```diff
@@ -41,9 +41,10 @@ export const StoreProduct: Record<string, Resolver<Root>> & {
   name: ({ isVariantOf, name }) => name ?? isVariantOf.productName,
   slug: ({ isVariantOf: { linkText }, itemId }) => getSlug(linkText, itemId),
   description: ({ isVariantOf: { description } }) => description,
-  seo: ({ isVariantOf: { description, productName } }) => ({
+  seo: ({ isVariantOf: { description, productName, linkText } }) => ({
     title: productName,
     description,
+    canonical: `/${linkText}/p`,
```

Comment:
I don't like this very much. This doesn't reflect the freedom we give to our users to choose how they're going to name their pages.

Reply:
not really, there are other parts that already force it, like the slug field etc